pom.xml

```diff
@@ -2,19 +2,18 @@
 <!-- $Id: pom.xml 39481 2011-05-12 20:14:54Z kutzi $ -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
+
   <parent>
-    <groupId>org.jvnet.hudson.plugins</groupId>
-    <artifactId>instant-messaging-parent</artifactId>
-    <version>1.29</version>
-    <relativePath>../instant-messaging-parent-plugin/pom.xml</relativePath>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>4.12</version>
   </parent>
```

Comment:
@timja : how much was this change needed about the parent? Can I revert this block?

Asking because I can't get the new release to publish, apparently old one went to "jvnet" dir tree and I have/had access there: https://repo.jenkins-ci.org/webapp/#/artifacts/browse/tree/General/releases/org/jvnet/hudson/plugins/ircbot/2.33 but the new attempt fails with 

````
    [INFO] --- maven-deploy-plugin:2.8.2:deploy (default-deploy) @ ircbot ---
    Uploading to maven.jenkins-ci.org: https://repo.jenkins-ci.org/releases/org/jenkins-ci/plugins/ircbot/2.34/ircbot-2.34.hpi
    Uploading to maven.jenkins-ci.org: https://repo.jenkins-ci.org/releases/org/jenkins-ci/plugins/ircbot/2.34/ircbot-2.34.pom
    [INFO] ------------------------------------------------------------------------
    [INFO] BUILD FAILURE
    [INFO] ------------------------------------------------------------------------
    [INFO] Total time: 32.998 s
    [INFO] Finished at: 2020-12-05T03:07:11+01:00
    [INFO] ------------------------------------------------------------------------
    [ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy
 (default-deploy) on project ircbot: Failed to deploy artifacts: Could not transfer artifact
 org.jenkins-ci.plugins:ircbot:hpi:2.34 from/to maven.jenkins-ci.org
 (https://repo.jenkins-ci.org/releases/):
 Access denied to: https://repo.jenkins-ci.org/releases/org/jenkins-ci/plugins/ircbot/2.34/ircbot-2.34.hpi,
 ReasonPhrase: Forbidden. -> [Help 1]
````

Which may make sense - at least the path is different now (jenkins-ci/plugins) and I likely don't have access there.

No idea what way forward is correct - fix back to jvnet namespace, or plead for common plugins location access? :)

Reply:
You can just re-add the groupid,

The reason I removed the parent was because it was quite dated and I didn’t really see the advantage in it